### DISCUSSION TITLE
Bring improvements from similar project

### DISCUSSION THREAD

**Discussion by Tarek-Hasan** [waydroid/waydroid#1462] [Ideas]
[KMRE (Kylin Mobile Runtime Environment)](https://gitee.com/openkylin/kmre) developed by OpenKylin, is an actively developed Anbox fork. Which has been since heavily developed and recently its closed sourced parts has been open sourced. I hope you would bring any relevant improvements and fixes from KMRE project to WayDroid.

Caveats:
KMRE's documentation still in fully Chinese. And its parts contain two different licenses, some Apache 2.0 and others GPL 3.0.

**Comment by Runa-Chin** [comment]
> [KMRE (Kylin Mobile Runtime Environment)](https://gitee.com/openkylin/kmre) developed by OpenKylin, is an actively developed Anbox fork. Which has been since heavily developed and recently its closed sourced parts has been open sourced. I hope you would bring any relevant improvements and fixes from KMRE project to WayDroid.
> 
> Caveats: KMRE's documentation still in fully Chinese. And its parts contain two different licenses, some Apache 2.0 and others GPL 3.0.

It's kinda still a lot of Chinese stuff right there. It looks like another free Android emulator we have seen for Windows. But I think they gonna make their own version of WayDroid
